include/config/config_debug.h

```diff
@@ -26,7 +26,26 @@
  * This allows you to quickly test the level you're working on.
  * If you want the game to boot normally, just comment out the define again.
  */
-// #define TEST_LEVEL LEVEL_BOB
+// #define STARTUP_SPAWN_LEVEL LEVEL_BOB
+
+/**
+ * Mario's starting area on boot.
+ * NOTE: Requires STARTUP_SPAWN_LEVEL.
+ * NOTE: It is also recommended to set STARTUP_SPAWN_POS with this, otherwise the default position may cause Mario to spawn out of bounds.
+ */
+// #define STARTUP_SPAWN_AREA 1
+
+/**
+ * Mario's starting position on boot.
+ * NOTE: Requires STARTUP_SPAWN_LEVEL.
+ */
+// #define STARTUP_SPAWN_POS 0, 0, 0
+
+/**
+ * Mario's starting yaw on boot.
```

Comment:
The 0x0 in the define makes this pretty clear already but still worth adding a mention that this is an s16 angle rather than an angle specified in degrees.